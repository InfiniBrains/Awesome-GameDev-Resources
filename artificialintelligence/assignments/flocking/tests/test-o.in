2.000 1.000 1.000 1.000 2.000 2.000 2.000 7
0.500 0.500 0.000 0.000
-0.500 -0.500 0.000 0.000
-0.500 0.500 0.000 0.000
0.500 -0.500 0.000 0.000
0.000 0.000 0.000 0.000
1.000 0.000 0.000 0.000
-1.000 0.000 0.000 0.000
256.000
