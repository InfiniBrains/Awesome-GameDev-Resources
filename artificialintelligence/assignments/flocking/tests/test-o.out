-5103.697 88588.336 -19.938 346.046
5103.697 -88588.336 19.938 -346.046
5103.697 88588.336 19.938 346.046
-5103.697 -88588.336 -19.938 -346.046
0.000 0.000 0.000 0.000
425341.933 0.000 1661.488 0.000
-425341.933 0.000 -1661.488 0.000
